src/containers/Network/NodeConfig.jsx

```diff
@@ -1,70 +1,60 @@
 import React, { Component } from 'react'
-import {
-  Button,
-  Input,
-  FileChooser,
-  Select,
-  NodeInfoBox
-} from 'ethereum-react-components'
-import './NodeConfig.css'
-import NodeSettingsForm from './NodeSettingsForm'
-import { Mist } from '../../API'
+import styled from 'styled-components'
+import { NodeSettings } from 'ethereum-react-components'
 
+const Mist = window.Mist
```

Comment:
This causes dev to break when not using the shell. The Mist API file takes care of using a mock API if no `window.Mist` is detected.